projects/storefrontlib/shared/components/carousel/carousel.component.ts

```diff
@@ -42,6 +42,7 @@ import { CarouselService } from './carousel.service';
   selector: 'cx-carousel',
   templateUrl: './carousel.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
+  host: { ngSkipHydration: 'true' },
```

Comment:
Future newcomers to this code might now know why we disabled hydration for this component.
Would you mind explaining in a code comment elaborating "what would happen if this code line wasn't there?